pkgs/by-name/de/deepsource/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  installShellFiles,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "deepsource";
+  version = "0.8.6";
+
+  nativeBuildInputs = [ installShellFiles ];
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
```

Comment:
This is not wrong but weird. Could you put this after `postInstall`?

Reply:
okey, on it @Aleksanaa

Reply:
Hi, @Aleksanaa fixed, could you please check again?